test/extensions/extensions_build_system.bzl

```diff
@@ -1,54 +1,66 @@
 load("//bazel:envoy_build_system.bzl", "envoy_benchmark_test", "envoy_cc_benchmark_binary", "envoy_cc_mock", "envoy_cc_test", "envoy_cc_test_binary", "envoy_cc_test_library", "envoy_py_test")
 load("@envoy_build_config//:extensions_build_config.bzl", "EXTENSIONS")
 
-def _apply_to_extension_allow_for_test(func, name, extension_name, **kwargs):
+def _apply_to_extension_allow_for_test(func, name, extension_name, additional_extension_names, **kwargs):
```

Comment:
Just change `extension_name` to `extension_names`?

Reply:
yeah actually I followed the suggestion by @antoniovicente https://github.com/envoyproxy/envoy/pull/17181#discussion_r660237245 wdyt?

Reply:
We'ld need to change all build files to package things in a list type unless we're willing to branch on argument type and accept extension_names both string and lists in extension_names.